src/components/ebay-breadcrumbs/test/test.server.js

```diff
@@ -16,37 +16,23 @@ describe('breadcrumbs', () => {
 
         expect(getByLabelText(input.a11yHeadingText)).has.attr('role', 'navigation');
 
-        input.items
-            .slice(0, -1)
-            .forEach((item) =>
-                expect(getByText(item.renderBody.text)).has.property('tagName', 'A')
-            );
-
-        expect(getByText(input.items[input.items.length - 1].renderBody.text)).has.property(
-            'tagName',
-            'BUTTON'
-        );
-    });
-
-    it('renders buttons when hrefs are missing', async () => {
-        const input = mock.Buttons;
-        const { getByText } = await render(template, input);
         input.items.forEach((item) =>
-            expect(getByText(item.renderBody.text)).has.property('tagName', 'BUTTON')
+            expect(getByText(item.renderBody.text)).has.property('tagName', 'A')
         );
     });
 
-    it('renders <button> for missing input href item', async () => {
-        const input = mock.Links_First_Without_HREF;
+    it('renders aria-current as location for last item without href', async () => {
+        const input = mock.Links_Last_Without_HREF;
         const { getByText } = await render(template, input);
 
-        input.items.slice(0, -1).forEach((item) => {
+        input.items.forEach((item) => {
             const itemEl = getByText(item.renderBody.text);
             if (item.href) {
                 expect(itemEl).has.property('tagName', 'A');
                 expect(itemEl).has.attr('href', item.href);
             } else {
-                expect(itemEl).has.property('tagName', 'BUTTON');
+                expect(itemEl).has.property('tagName', 'A');
+                expect(itemEl).attr('aria-current', 'location');
```

Comment:
Actually this test is somewhat invalid. I would expect that `aria-current` to be ONLY on the last one, and not on any items without a `href`